reproc/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ reproc_test(reproc overflow C)
 reproc_test(reproc path C)
 reproc_test(reproc stop C)
 reproc_test(reproc working-directory C)
+reproc_test(reproc pid C)
```

Comment:
This needs to be UNIX only (see the if just after this) because fork isn't supported on Windows.